README.md

```diff
@@ -6,25 +6,118 @@ Metrilo helps you track the events that the people visiting your shop do - viewi
+
+```
+curl -X POST https://trk.mtrl.me/category" -H "Content-Type: text/plain" -d \
+'{
+  "time": 1518004715732,
+  "token": "sh0p-t0k3n",
+  "params": {
+    "id": "1",
+    "name": "Awesome clothing",
+    "url": "https://my-awesome-shop.com/awesome-clothing"
+  }
+}'
+```
+
+```
+curl -X POST https://trk.mtrl.me/product" -H "Content-Type: text/plain" -d \
+'{
+  "time": 1518004715732,
+  "token": "sh0p-t0k3n",
+  "params": {
+    "id": "2",
+    "categories": ["1"],
+    "name": "Awesome T-Shirt",
+    "url": "https://my-awesome-shop.com/awesome-clothing/awesome-t-shirt"
+  }
+}'
+```
+
+```
+curl -X POST https://trk.mtrl.me/customer" -H "Content-Type: text/plain" -d \
+'{
+  "time": 1518004715732,
+  "token": "sh0p-t0k3n",
+  "params": {
+    "email": "johnybravo@gmail.com",
+    "createdAt": 1518004715732
+  }
+}'
+```
+
+The next step is to install the frontend library. Add the following snippet to your website (preferably the `<head>` tag):
+
+```
+<script type="text/javascript" src="https://trk.mtrl.me/tracking.js?token=sh0p-t0k3n"></script>
+```
+
+When you load that script, you should have access to a `window.metrilo` object. That's what you're going to use to create tracking events on behalf of your customers.
+
+Now let's create the following user journey - the user opened your homepage, navigated to the category page, then the product, then added it to the cart and finally placed an order. Here is what you would do on the frontend:
+
+```javascript
+window.metrilo.viewPage('http://my-awesome-shop.com/', { name: 'Awesome homepage' }) // This would be called on your homepage
+window.metrilo.viewCategory('1') // This would be called on the 'Awesome clothing' category page
+window.metrilo.viewProduct('2') // This would be called on the 'Awesome T-Shirt' product page
+window.metrilo.addToCart('2') // This would be called when a user clicks an add to cart button
+window.metrilo.checkout() // This would be called whenever the customer starts the checkout process
+```
+
+At that point, all of the actions are done by an anonymous user (you can check that in the Metrilo app) - you can either ask them to login now and use the email they provided:
+
+```javascript
+window.metrilo.identify('johnybravo@gmail.com')
+```
+
+...or wait until they fill their email inside your order details form. In any case, when they finish the order, you'll need to send the order event from your backend:
+
+```
+curl -X POST https://trk.mtrl.me/order" -H "Content-Type: text/plain" -d \
+'{
+  "time": 1518004715732,
+  "token": "sh0p-t0k3n",
+  "params": {
+    "id": "314",
+    "createdAt": "1518004715732",
+    "email": "johnybravo@gmail.com",
+    "amount": "10.42",
+    "status": "completed",
+    "products": [
+      "productId": "2",
+      "quantity": 1
+    ]
+  }
+}'
+```
+
+That's it! Now when all events are processed by Metrilo, they will be attributed to the `johnybravo@gmail.com` user.
```

Comment:
This seems like a good place to add a screenshot of the specific customer's session in the CRM.

Reply:
I can try 👍

Reply:
I've decided against putting a screenshot here - a screenshot at that point would ask much more questions than it would answer - especially regarding sessions, timestamps, and identify. I just want to keep the quick guide shorter and let these questions be answered in the API reference. Let me know if you agree with that